portfolio/src/components/Cover.tsx

```diff
@@ -1,36 +0,0 @@
-import React, { Component } from 'react';
-import profile from "../images/profile.png";
-import Container from 'react-bootstrap/Container';
-
-type Props = {
-
-}
-export default class Cover extends Component<Props>{
-    constructor(props: Props) {
-        super(props);
-    }
-    getProfile() {
-        const backgroundStyles = {
-            backgroundImage: `url(${
-                "https://i.imgur.com/K2zj5sk.png"
-                })`,
-            backgroundSize: '15rem'
-        }
-        return backgroundStyles;
-    }
-    render() {
-        return (
-            <Container fluid={true} className="cover-container">
-                <div className="cover-content">
-                    <div className="cover-photo">
-                        <div className="profile" style={this.getProfile()}></div>
```

Comment:
getprofile is static css, not reliant upon state. I would just have this in CSS and call it from there as a class instead, because CSS has performance benefits over JS in this case. (minute, but still)

Reply:
You would want to use something like this if you had something that updated this cover component a lot and the state updates based on that